### PR TITLE
manual: update tonecurve for darktable 2.6

### PR DESCRIPTION
**Pull request by aurelienpierre**
full disclosure : it's the first time I play with the manual and I haven't tried to build it.

### CODE REVIEW

File: doc/usermanual/darkroom/modules/tone/tonecurve.xml [hunk 1]
```diff
@@ -68,11 +67,74 @@
       below) at the selected spot or area are shown on top left of the widget.
     </para>
 
+    <sect5 status="draft">
+      <title>color spaces</title>
+      <para>
+        Depending on the desired intent, you car apply the tone curve in 3 different color spaces:
+        <ul>
```
Comment (upegelow) [inline doc/usermanual/darkroom/modules/tone/tonecurve.xml]: Please use <itemizedlist></itemizedlist> and <listitem></listitem>.

Comment (aurelienpierre) [reply doc/usermanual/darkroom/modules/tone/tonecurve.xml]: done

File: doc/usermanual/darkroom/modules/tone/tonecurve.xml [hunk 2]
```diff
@@ -68,11 +67,74 @@
       below) at the selected spot or area are shown on top left of the widget.
     </para>
 
+    <sect5 status="draft">
+      <title>color spaces</title>
+      <para>
+        Depending on the desired intent, you can apply the tone curve in 3 different color spaces:
+        <itemizedlist>
+          <listitem>Lab, linked channels,</listitem>
```
Comment (upegelow) [inline doc/usermanual/darkroom/modules/tone/tonecurve.xml]: Oops, xmllint still complaining. The content of each listitem needs to be enclosed in `<para></para>`.

Comment (aurelienpierre) [reply doc/usermanual/darkroom/modules/tone/tonecurve.xml]: it should be good now

**Comment by rawfiner** [comment]
This sentences sound a bit contradictory to me, though I do not have the knowledge to check this, could you please check them? 
"XYZ is a linear technical color space designed to link the physiologic light response of human eyes to RGB spaces."
"it does so in a way that does NOT account for the role of the brain correction in human perceptions."

**Comment by aurelienpierre** [comment]
XYZ derivates from the physiological cone response of the CIE standard observer (the sensors in the eyes). In block diagram that would translates:

( spectral vector : wavelengths, intensities ) -> [ eyes ] -> ( tristimulus vector : intensities ) -> [ brain ] -> (perception)

The (classic, old, flawed, inaccurate) mathematical model to describe that is :
( spectral vector or camera RGB ) -> [ XYZ conversion ] -> ( tristimulus vector : intensities ) -> [ Lab conversion ] -> (perception)

**Comment by rawfiner** [comment]
Ok, that makes sense :-)
Thanks for the explanation !

**Comment by upegelow** [comment]
Thanks, I think it's well explained. Some proof reading to be done later.

**Pull request merged**